crates/matrix-sdk/src/event_cache/mod.rs

```diff
@@ -702,7 +702,7 @@ pub enum RoomEventCacheUpdate {
     Clear,
 
     /// The fully read marker has moved to a different event.
-    UpdateReadMarker {
+    ReadMarker {
```

Comment:
I disagree with this change: this breaks symmetry with the other two variants which start with a verb, and since it's a deferred command it makes sense to start with a verb. Can you revert this please?

Reply:
`…Update::Update…`, that's what I dislike. What do you think about `MoveReadMarker`?

Reply:
Sounds good!